authors/settings.py

```diff
@@ -81,8 +81,13 @@
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
+        'ENGINE': 'django.db.backends.postgresql_psycopg2',
+        'NAME': 'auths_heaven',
+        'USER': 'postgres',
+        'PASSWORD': 'DB_PASS',
+        'HOST': 'localhost',
+        'PORT': '5432',
```

Comment:
We do not want to expose these details in our repo. Create a .env file and hide them there (all the keys except ENGINE). Let user know they need to add these keys (with values) to their own .env file, so you won't need to tell them what NAME or USER or PORT to use. 
All user needs to do is set up a Postgres database and add their details to their .env file. `.env` should be gitignored

Reply:
Well noted.